### PR TITLE
doc/types-modules-offline-exporting

### PR DESCRIPTION
**Pull request by bre1470**
TypeScript-compatible doclets.

### CODE REVIEW

File: js/modules/offline-exporting.src.js [hunk 1]
```diff
@@ -6,11 +6,54 @@
  * License: www.highcharts.com/license
  */
 
+/**
+ * Download options for offline exporting.
+ *
+ * @typedef Highcharts.OfflineExportingDownloadOptionsObject
```
Comment (TorsteinHonsi) [inline js/modules/offline-exporting.src.js]: I believe these are the same options that are available for the online exporting, so they should probably use the same typedef.

File: js/modules/offline-exporting.src.js [hunk 2]
```diff
@@ -6,11 +6,54 @@
...
+ *
+ * @property {number} [scale]
+ *           Scaling factor of downloaded image compared to source
+ *
+ * @property {string} [type]
+ *           File type of resulting download
+ */
+
+/**
+ * @typedef Highcharts.OfflineExportingOptionsObject
```
Comment (TorsteinHonsi) [inline js/modules/offline-exporting.src.js]: I believe these are the same options that are available for the online exporting, so they should probably use the same typedef.

**Comment by sophiebremer** [comment]
I changed the dependencies.

File: js/modules/offline-exporting.src.js [hunk 3]
```diff
@@ -6,11 +6,34 @@
  * License: www.highcharts.com/license
  */
 
+/**
+ * Download options for offline exporting.
+ *
+ * @interface Highcharts.OfflineExportingOptionsObject
```
Comment (TorsteinHonsi) [inline js/modules/offline-exporting.src.js]: I don't think we need a specific options object for offline exporting. The `libURL` option is the only thing missing in online, while `scale` already exists in online. We should rather add these to the `Highcharts.ExportingOptionsObject`.

**Comment by bre1470** [comment]
I removed the interfaces.

**Pull request merged**